addons/mail/static/src/components/chatter/chatter.js

```diff
@@ -51,6 +51,8 @@ class Chatter extends Component {
          * Reference of the message list. Useful to trigger the scroll event on it.
          */
         this._threadRef = useRef('thread');
+        this._scrollPanelRef = useRef('scrollPanel');
```

Comment:
Alphabetical order (before `_threadRef`) and also add a docstring to say what it contains and why we need it.